### PR TITLE
Fix the facet responses to avoid duplicate entries

### LINKED ISSUE #651: /api/submissions facets have duplicate entries for bootcamps
Example:

```
    "facets": {
        "review_statuses": [
            {
                "review_status": "approved",
                "count": 3
            },
            {
                "review_status": "pending",
                "count": 1
            },
            {
                "review_status": "rejected",
                "count": 2
            }
        ],
        "bootcamps": [
            {
                "bootcamp_id": 1,
                "bootcamp_title": "Coffee Baths for Cats",
                "count": 1
            },
            {
                "bootcamp_id": 2,
                "bootcamp_title": "The Best Cats Love Tuna",
                "count": 1
            },
            {
                "bootcamp_id": 2,
                "bootcamp_title": "The Best Cats Love Tuna",
                "count": 1
            },
            {
                "bootcamp_id": 2,
                "bootcamp_title": "The Best Cats Love Tuna",
                "count": 1
            },
            {
                "bootcamp_id": 2,
                "bootcamp_title": "The Best Cats Love Tuna",
                "count": 1
            },
            {
                "bootcamp_id": 2,
                "bootcamp_title": "The Best Cats Love Tuna",
                "count": 1
            }
        ]
    }
```

### PR DESCRIPTION
**Pull request by rhysyngsun**
#### What are the relevant tickets?
Fixes #651 

#### What's this PR do?
This alters the query for the bootcamps facet in order to ensure there are no duplicate entries

#### How should this be manually tested?
- Ensure you have several applications for a given bootcamp
- Go to /api/submissions/ and ensure the facets reflect the state correctly with no duplicates

### CODE REVIEW

**Comment by aliceriot** [comment]
looks great!

Screenshot from 2020-06-16 09-39-10

File: applications/views.py [hunk 1]
```diff
@@ -111,18 +111,22 @@ def get_facets(self, queryset):
...
-                bootcamp_id=F("bootcamp_application__bootcamp_run__bootcamp__id"),
-                bootcamp_title=F("bootcamp_application__bootcamp_run__bootcamp__title"),
-                count=Count("bootcamp_application"),
-            )
-            .values("bootcamp_id", "bootcamp_title", "count")
+            Bootcamp.objects.values("id", "title")
+            .annotate(count=Subquery(qs, output_field=IntegerField()))
+            .filter(count__gte=1)
         )
+        print(bootcamps.query)
```
Comment (aliceriot) [inline applications/views.py:129]: stray print?

Comment (rhysyngsun) [reply applications/views.py:129]: d'oh

**Comment by rhysyngsun** [comment]
@aliceriot fixed and rebased

**Comment by aliceriot** [comment]
looks like there might be a test issue?

**Comment by rhysyngsun** [comment]
@aliceriot all set now

**Pull request merged**